pkg/ocicni/ocicni.go

```diff
@@ -173,13 +183,10 @@ func InitCNI(pluginDir string, cniDirs ...string) (CNIPlugin, error) {
 		return nil, err
 	}
 
-	// Fail loudly if plugin directory doesn't exist, because fsnotify watcher
-	// won't be able to watch it.
-	if _, err := os.Stat(pluginDir); err != nil {
-		return nil, err
-	}
-
-	go plugin.monitorNetDir()
+	go func() {
+		plugin.monitorNetDir()
+		time.Sleep(monitorNetDirPeriod)
+	}()
```

Comment:
Did you intend for there to be a loop around these two? I don't see what it is trying to achieve otherwise.

Reply:
My bad... Will fix.